packages/react-mutation-mapper/package.json

```diff
@@ -37,6 +37,7 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
+    "@types/react-table": "^6.8.7",
```

Comment:
It might be better to add this to the root package.json, since we already moved all `@types` dependencies there. Having `@types` in multiple packages is problematic.